### PR TITLE
update the sprint template

### PR DESCRIPTION
**Pull request by diasdavid**
main change is add links to discussion pad for every endeavour sprint and explicitly appoint the lead for that endeavour (person that should be always in the sprint and lead the discussion)

### CODE REVIEW

File: sprint-issue-template.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+## Sprint Discussions
+
+[Schedule](https://github.com/ipfs/pm#sprint-discussion-schedule)
+
+Please take notes in a separate pad, if you can, and link it here.
+
+Endeavour         | Lead           | Pad    | Permanent Link
+:---------------: | :------------: | :----: |
+Apps on IPFS      | Juan Benet     |        |
```
Comment (jbenet) [inline sprint-issue-template.md]: maybe we can break this up into: 
- Station + Webui led by @dignifiedquire
- Apps on IPFS  led by me
?

File: sprint-issue-template.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+infrastructure    | Lars Gierth    |        |
+libp2p            | David Dias     |        |
+node-ipfs         | David Dias     |        |
+go-ipfs           | Jeromy Johnson |        |
+testing + ci      | Jeromy Johnson |        |
+protocol + specs  | Juan Benet     |        |
+bitswap ml        | Juan Benet     |        |
+Data Structures   | Juan Benet     |        |
```
Comment (jbenet) [inline sprint-issue-template.md]: prob make the names be github handles

File: sprint-issue-template.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+- 
+
+## Sprint Discussions
+
+[Schedule](https://github.com/ipfs/pm#sprint-discussion-schedule)
+
+Please take notes in a separate pad, if you can, and link it here.
+
+Endeavour         | Lead           | Pad    | Permanent Link
+:---------------: | :------------: | :----: |
```
Comment (jbenet) [inline sprint-issue-template.md]: the form doesn't render right-- isnt a gfm form i think

File: sprint-issue-template.md (hunk 1 above)
Comment (Dignifiedquire) [inline sprint-issue-template.md]: Sounds good, though I think one time slot for both would be enough for now and if it expands we can split it into two time slots.

Comment (jbenet) [inline sprint-issue-template.md]: ok sounds good

File: sprint-issue-template.md (hunk 2 above)
Comment (Dignifiedquire) [inline sprint-issue-template.md]: IRC + github + realname, or at least have a mapping somewhere it's pretty hard at times to map people together if they don't use the same handle everyhwere

**Comment by RichardLitt** [comment]
Shouldn't this replace `sync.md` in this repo?

**Comment by RichardLitt** [comment]
LGTM!

**Comment by diasdavid** [comment]
thank you :)

**Pull request merged**